src/baldr/compression_utils.cc

```diff
@@ -0,0 +1,125 @@
+#include "baldr/compression_utils.h"
```

Comment:
so we had some code already in skadi for doing decompression of zipped elevation sources but it was very specific to that use case. i reimplemented this in a generic way so that it could be use not only by skadi but also by baldr for compressed tile file reading (which was using boost::iostreams)